fs_overlay/var/lib/nginx-conf/nginx.conf.erb

```diff
@@ -24,16 +24,40 @@ http {
                       '"$http_user_agent" "$http_x_forwarded_for"';
     <% end %>
 
-    access_log <%= ENV['ACCESS_LOG'] || "off" %>;
-
     <% if ENV['WEBSOCKET'] && ENV['WEBSOCKET'].downcase == 'true' %>
     map $http_upgrade $connection_upgrade {
         default upgrade;
         ''      close;
     }
     <% end %>
 
-    access_log  /var/log/nginx/access.log  main;
+    <% if ENV['ACCESS_LOG'] == 'off' || ENV['ACCESS_LOG'] == '' %>
+    access_log off;
+    <% elsif ENV['ACCESS_LOG'] == 'stdout' %>
+    access_log /dev/stdout main buffer=<%= ENV['ACCESS_LOG_BUFFER'] || "16k" %>;
+    <% else %>
+    <% elsif ENV['ACCESS_LOG'] == 'stderr' %>
+    access_log /dev/stderr main buffer=<%= ENV['ACCESS_LOG_BUFFER'] || "16k" %>;
```

Comment:
I don't think having buffer by default is a good idea. People will be confused since nothing is mentioned in the documentation.